### PR TITLE
Add redirects to `stable` docs

### PR DESCRIPTION
**Pull request by ajschmidt8**
This PR adds a redirect for each of our libraries so that URLs like `/api/cudf/` redirect to `/api/cudf/stable/` instead of showing a 404 page.

**Note**: The `/api` pages won't actually show in PR preview environments currently.

This will be resolved as part of our planned future enhancements for our websites.

### CODE REVIEW

**Comment by ajschmidt8** [comment]
Navigating to https://deploy-preview-401--docs-rapids-ai.netlify.app/api/cudf redirects to https://deploy-preview-401--docs-rapids-ai.netlify.app/api/cudf/stable// (note the double trailing `//`).

I think this is because there are no files in that path for PR previews.

I think we can ignore that and expect it to work correctly for the production deployment since our production deployment pulls in the documentation files from S3 (see this workflow: https://github.com/rapidsai/docs/blob/main/.github/workflows/deploy.yaml).

File: _redirects [hunk 1]
```diff
@@ -0,0 +1,40 @@
+/api/cucim /api/cucim/stable/
+/api/cucim/ /api/cucim/stable/
+/api/cudf /api/cudf/stable//
```
Comment (bdice) [inline _redirects:3]: @ajschmidt8 Was the issue you saw in this comment https://github.com/rapidsai/docs/pull/401#issuecomment-1589550633 just a typo?
```suggestion
/api/cudf /api/cudf/stable/
```

**Review by bdice — CHANGES_REQUESTED** [review]
LGTM -- one question.

File: _redirects (hunk 1 above)
Comment (ajschmidt8) [reply _redirects:3]: ohhhhh shoot. yes. most likely.

i was doing bulk updates via some fancy VS Code keyboard shortcuts and they must've gone awry.

let me update this and try again.

good catch. this is why we do code reviews :slightly_smiling_face:

**Pull request merged**